### ISSUE TITLE
Export CSV feature doesn't work correctly

### ISSUE THREAD

**Issue by chrisdolby-HE** [microsoft/vsts-extension-retrospectives#126]
Hi there,

I would have thought that this issue has come up before, but exports from the Retrospectives extension are in plain UTF-8 with only simple comma separation and no containing delimiters for, you know, sentences.

Please could someone look at adding in more cleanly exportable options, even Excel if need be? CSVW is also an option but probably a lot of extra weight and complexity.

**Comment by emerald313** [comment]
Yes, came across this today too. Comma's in sentences break the exported CSV

**Comment by cujo-msft** [comment]
Same issue here, can you advise when this will be worked?

**Comment by rwhitworth127** [comment]
Same, I originally brought up the idea of having an export to csv feature. The work happened fairly quickly, but I saw the same problem, commas in sentences break the export. 
I am not sure what the easiest path forward is here, but fixing csv, changing to tab formatted, or Excel (xlsx) would be great.

**Comment by mindlessroman** [comment]
@kaskadz - I believe this was handled in a recent release. Can you confirm?
